src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -113,11 +114,6 @@ export default function Swap({
   // Cow subsidy modal
   const openCowSubsidyModal = useOpenModal(ApplicationModal.COW_SUBSIDY)
   const showCowSubsidyModal = useModalIsOpen(ApplicationModal.COW_SUBSIDY)
-  // Native wrap modals
-  const [showNativeWrapModal, setOpenNativeWrapModal] = useState(false)
-  const openNativeWrapModal = () => setOpenNativeWrapModal(true)
-  const dismissNativeWrapModal = () => setOpenNativeWrapModal(false)
-
```

Comment:
@W3stside relevant to your latest changes, can you also take a look?

Reply:
And please, check the final code in the branch: https://github.com/cowprotocol/cowswap/tree/refactor/new-swap-form-1
Because I have changed this file a lot in the next PR's